build/templates/_attributes.py.mako

```diff
@@ -104,6 +104,26 @@ class AttributeEnum(object):
         return self._underlying_attribute.__set__(session, value.value)
 
 
+class AttributeEnumWithConverter(AttributeEnum):
+    def __init__(
```

Comment:
needs docstring

Reply:
I have added docstrings for `AttributeEnumWithConverter` class and its constructor in the latest commit